app/src/main/java/ch/epfl/sdp/appart/scrolling/card/Card.java

```diff
@@ -2,19 +2,35 @@
 
 import javax.annotation.Nullable;
 
+/**
+ * Object to represent a tile in the scrolling activity.
+ * <p>
+ * It contains a subset of information related to an ad. The card ID refers to the ID of the
+ * Firestore document containing data of the card.
+ */
 public class Card {
+    private final String adId;
     private final String id;
     private final String ownerId;
     private String city;
     private long price;
     private String imageUrl;
     private boolean hasVRTour;
 
-    /*
-     * A newly created card will be local, hence the id won't be assigned (null). At the first
-     * upload to Firestore, a new unique id will be generated and assigned to the card.
+    /**
+     * Constructor of a Card.
+     * <p>
+     * The id is nullable because a locally created card will need the upload to Firestore to
```

Comment:
With the new implementation of putAd, the Card related to an Ad is created after the Ad itself so the Card always have an ID atcreation. I will remove the @Nullable in the constructor before merging the corresponding PR.